style.css

```diff
@@ -507,3 +518,157 @@ section {
   background-color: #fff;
   border-radius: 8px;
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    overflow-x: hidden;
+  }
+
+  #menu {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    visibility: visible;
+    gap: 32px;
+    position: relative;
+    bottom: 15px;
+    right: 10.76%;
+  }
+
+  .nav-links a {
+    width: 46px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .navigation,
+  .bio-section,
+  .about-section,
+  .contact-me {
+    width: 100vw;
+    overflow-x: hidden;
+  }
+
+  .bio-section {
+    height: 100vh;
+    position: relative;
+    top: -80px;
+    background-image: url(img/desk-bg.png);
+    background-size: 100% auto;
+  }
+
+  .bio-section .content {
+    padding: 10%;
+    width: 80vw;
+    position: relative;
+    top: 4%;
+  }
+
+  .bio-header {
+    width: 44vw;
+  }
+
+  .bio-text {
+    font-size: 1rem;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: column;
+    gap: 88px;
+  }
+
+  .card {
+    width: 80.2vw;
+    height: 70%;
+    position: relative;
+    left: 9.86%;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .card-img img {
+    height: 172%;
+    width: 102%;
+    position: relative;
+    top: -39%;
+    left: -5%;
+  }
+
+  .card-items {
+    width: 99%;
+    height: 81%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 12%;
+    position: relative;
+    left: 4%;
+    top: -16%;
+  }
+
+  .card-props {
+    margin-right: 7%;
+  }
+
+  .oppo {
+    left: 88%;
+  }
+
+  #oppo,
+  #oppo2 {
+    position: relative;
+    left: 1%;
+    top: -15%;
+  }
+
+  .about-section .content {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    margin-top: 21%;
+  }
+
+  .about-2 {
+    position: relative;
+    top: 10%;
+    gap: 21%;
+  }
```

Comment:
Please I'd suggest you gradually work on the about-section of your project to be as close as possible to the Figma template, your design looks quite different from the alignment and the design we have 😄 

Figma design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/b6d33799-a580-493f-b36e-a4e1bdc3dbf2)

Your design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/b7eaffc3-1a80-40cd-8dd4-597dc65888fd)